components/org.wso2.carbon.identity.oauth.endpoint/src/main/java/org/wso2/carbon/identity/oauth/endpoint/util/EndpointUtil.java

```diff
@@ -963,6 +965,20 @@ private static void setConsentRequiredScopesToOAuthParams(AuthenticatedUser user
                 }
             }
             if (CollectionUtils.isNotEmpty(allowedOAuthScopes)) {
+                // filter out internal scopes to be validated
```

Comment:
Please make sure all the comments adhering following rules.
1. Comments should start with capital letter
2. Comments should end with a full stop.
3. Comments should start with a space.

Reply:
Sure. I'll do the necessary changes

Reply:
This has been fixed with https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/1766/commits/ca0b0b94f26a3d0f66fc9d3e6b0ba33c7dd8075b